styles.css

```diff
@@ -120,9 +128,213 @@ body {
   opacity: 0.5;
 }
 
-@media (min-width: 786px) {
+/* Work  */
+.work {
```

Comment:
- [ ] You did a great job with the design. :+1: Let's work on more details to meet the project requirements. Kindly use `grid` for the container holding the entire cards and please use `flex` for the individual cards. That way it'll be easier when working on the desktop version of the page. 🥸️

**Requirements**
![Screenshot from 2023-07-05 09-19-05](https://github.com/evansnyamekye/Post-index/assets/17243325/013fd7bb-62ec-4441-adcd-19ffe593bb1f)